### ISSUE TITLE
In wide wide screen, search box goes too wide and pushes other elements into next row

### ISSUE THREAD

**Issue by Xymmer** [SAP/spartacus#7828]
The following happens if you are using a wide screen, and your window is beyond 2700 or 2800 pixels.
Search box gets % wide enough to knock the cart and sign in (eventually) to next line.

![image](https://user-images.githubusercontent.com/4998145/83441244-355fe680-a414-11ea-8c55-690e0f94a04a.png)

**Comment by Xymmer** [comment]
as we're trying to get 2.1 out the door, moving to 3.0
so don't merge to develop until 2.1-maintenance is created

**Comment by developpeurweb** [comment]
I left some comments in the PR: #8218

**Comment by bgambocjaviniar** [comment]
moving to do because owner for the PR finished their internship,
**therefore**, anyone can pick up or create a new PR.

**Comment by developpeurweb** [comment]
Fixed now, emulated to a 5120 resolution, being the top end of resolutions for consumer PC screens as of 2020.

![Screen Shot 2020-10-19 at 5 20 12 PM](https://user-images.githubusercontent.com/15113219/96513044-72940c00-122f-11eb-9fa1-6cda3196ee54.png)

**Comment by saad-mo** [comment]
Just a small [question/comment](https://github.com/SAP/spartacus/pull/9412#issuecomment-712484647) on the PR.

**Comment by saad-mo** [comment]
Tested on latest develop on local and works as expected 

![Screen Shot 2020-10-21 at 11 12 01 AM](https://user-images.githubusercontent.com/15215400/96747753-4ba01c00-138e-11eb-9724-19a62927fafe.png)
